EIPS/eip-4844.md

```diff
@@ -307,45 +343,16 @@ def get_intrinsic_gas(tx: SignedBlobTransaction, pre_state: ExecState) -> int:
     intrinsic_gas += len(tx.message.blob_versioned_hashes) * get_blob_gas(pre_state)
     return intrinsic_gas
 
-def get_blob_gas(pre_state: ExecState) -> int:
-    blocks_since_start = get_current_block(pre_state) - FORK_BLKNUM
-    targeted_total = blocks_since_start * TARGET_BLOB_TXS_PER_BLOCK
-    actual_total = read_storage(
-        pre_state,
-        SYSTEM_STATE_ADDRESS,
-        TOTAL_BLOB_TXS_STORAGE_SLOT
-    )
-    if actual_total < targeted_total:
+def get_blob_gas(header: Header) -> int:
+    if header.blobs == 0:
         return 0
```

Comment:
I don't think we need to introduce this special case, in the old version 0 was just passed into the `fake_exponential` function normally.